pandas/tests/indexes/period/test_constructors.py

```diff
@@ -6,35 +6,42 @@
 from pandas.core.dtypes.dtypes import PeriodDtype
 
 import pandas as pd
-from pandas import Index, Period, PeriodIndex, Series, date_range, offsets, period_range
+from pandas import (
+    Index,
+    NaT,
+    Period,
+    PeriodIndex,
+    Series,
+    date_range,
+    offsets,
+    period_range,
+)
 import pandas._testing as tm
 from pandas.core.arrays import PeriodArray
 
 
 class TestPeriodIndex:
     def test_construction_base_constructor(self):
         # GH 13664
-        arr = [pd.Period("2011-01", freq="M"), pd.NaT, pd.Period("2011-03", freq="M")]
-        tm.assert_index_equal(pd.Index(arr), pd.PeriodIndex(arr))
-        tm.assert_index_equal(pd.Index(np.array(arr)), pd.PeriodIndex(np.array(arr)))
+        arr = [Period("2011-01", freq="M"), NaT, Period("2011-03", freq="M")]
+        tm.assert_index_equal(Index(arr), PeriodIndex(arr))
```

Comment:
ideally make these 
res = 
exp =

Reply:
agreed.  per request from @WillAyd, trying to keep pure-moves separate from other cleanups.  dont worry, there's plenty more index test organization on its way